### ISSUE TITLE
decentralisation

### ISSUE THREAD

**Issue by kigerpunk** [tildeclub/tilde.club#58]
tilde.club is no longer the only tildebox (I think that’s what I’m going to call them.), nor should it be the “main” one. It’s the first and largest, but if this concept is going to work, we need to get the others up to scale to some extent.

A lot of this is that the others don’t have enough people, but it seems that with 6000 people on the tilde.club waitlist, totallynuclear.club (with the second most members) should have more than 46 members. This applies just as well to the other tildeboxes.

For now, tilde.club is the de-facto leader of the others. While it makes sense to keep box-specific repos and issues to their respective GitHub repos, I’d recommend moving the more general stuff to @metatilde. If @ftrain is willing, it might be best to move some of @tildeclub’s repos to @metatilde.

In summary:
- If you’re on the tilde.club waitlist, consider applying to totallynuclear.club (and the others once they automate the process - I don’t want to overwhelm them).
- If you own/rent a tildebox, put your box’s code in a GitHub repo/organisation.
- If you want to help the tilde movement keep developing, comment on issues that interest you. (I suppose I don’t need to say that to those of you who are already here.)
